### PR TITLE
INitial commit

### PR DESCRIPTION
**Pull request by NTKBatch5**
IMplemented US123 automation

### CODE REVIEW

**Comment by Massik84** [comment]
good job

**Comment by student-ap** [comment]
Great work

File: src/demo/DemoTest.java [hunk 1]
```diff
@@ -12,7 +12,7 @@ public static void main(String[] args) {
 		System.out.println(num); //added this line
 		
 		//from gitHUB
-		//Batch 7
+		//Batch 7
```
Comment (bashiraPoya) [inline src/demo/DemoTest.java:15]: Can format the code please...

**Comment by fdacunha22** [comment]
Well done.

**Pull request merged**